### PR TITLE
[Trivia] Handle potential Discord errors in session

### PR DESCRIPTION
**Pull request by PredaaA**
### Description of the changes
This PR handle NotFound or Forbidden errors in Trivia's session wait_for_answer method.

### CODE REVIEW

File: redbot/cogs/trivia/session.py [hunk 1]
```diff
@@ -207,7 +208,8 @@ async def wait_for_answer(self, answers, delay: float, timeout: float):
             )
         except asyncio.TimeoutError:
             if time.time() - self._last_response >= timeout:
-                await self.ctx.send(_("Guys...? Well, I guess I'll stop then."))
+                with contextlib.suppress(discord.HTTPException):
+                    await self.ctx.send(_("Guys...? Well, I guess I'll stop then."))
```
Comment (Jackenmen) [inline redbot/cogs/trivia/session.py]: Could you send the traceback that you are getting here that isn't a Forbidden nor NotFound?

Comment (PredaaA) [reply redbot/cogs/trivia/session.py]: Yeah I don't know why I have set HTTPException here, just changed.

**Review by Jackenmen — COMMENTED** [review]
see below

**Review by Jackenmen — CHANGES_REQUESTED** [review]
Hmm, there are a lot of places where we send messages and might possibly run into such issue, perhaps it would be better to instead just add this to `_error_handler()` method?

**Comment by PredaaA** [comment]
> Hmm, there are a lot of places where we send messages and might possibly run into such issue, perhaps it would be better to instead just add this to `_error_handler()` method?

Good point, thanks for pointing it out! That's something I didn't saw.
I've added those to `_error_handler` method on latest commit.

**Review by Jackenmen — APPROVED** [review]
Seems like this should just work 😏

**Pull request merged**